FQA update (#6506)
harness/developer-hub: kb/platform/harness-platform-faqs.md

"Exit code 137" typically indicates an out-of-memory error. When a process in a system exhausts its allocated memory resources, the operating system sends a termination signal to the process. In the case of "Exit code 137," this signal signifies that the process was terminated due to running out of memory. This error commonly occurs when a program or container attempts to allocate more memory than is available, leading to termination by the system to prevent resource exhaustion and potential system instability.

> ### How to install latest version of azure cli in our Harness Dlegates.

Reviewer: ```suggestion
### How to install the latest version of Azure CLI in Harness Delegate?
```

> You can follow:

Reviewer: ```suggestion
* Locate the `INIT_SCRIPT` in the `Deployment` object of the `delegate.yaml` file.
   ```
   - name: INIT_SCRIPT  
   value: ""  
   ```
* Replace `value: "" ` with the following:
```

```
## Install Azure CLI
rpm --import <https://packages.microsoft.com/keys/microsoft.asc>
rpm -ivh <https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm>
microdnf install -y azure-cli

> ```

Reviewer: Shouldn't you mention locate the `INIT_SCRIPT` in the `delegate.yaml` and replace value"" with the following microdnf script? Something like this, https://developer.harness.io/docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/tanzu-app-services-quickstart#install-cloud-foundry-command-line-interface-cf-cli-on-your-harness-delegate.

Reviewer 2: @roshnisarangadharan I removed this a a duplicate question. It is already covered in the Delegate section.

> 
> ```
> ## Install Azure CLI
> rpm --import <https://packages.microsoft.com/keys/microsoft.asc>
> rpm -ivh <https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm>
> microdnf install -y azure-cli
> 
> ```

Reviewer: ```suggestion
* Locate the `INIT_SCRIPT` in the `Deployment` object of the `delegate.yaml` file.
   ```
   - name: INIT_SCRIPT  
   value: ""  
   ```
* Replace `value: "" ` with the following:

  ```
  ## Install Azure CLI
  rpm --import <https://packages.microsoft.com/keys/microsoft.asc>
  rpm -ivh <https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm>
  microdnf install -y azure-cli
  ```
```

"Exit code 137" typically indicates an out-of-memory error. When a process in a system exhausts its allocated memory resources, the operating system sends a termination signal to the process. In the case of "Exit code 137," this signal signifies that the process was terminated due to running out of memory. This error commonly occurs when a program or container attempts to allocate more memory than is available, leading to termination by the system to prevent resource exhaustion and potential system instability.

> ### How to install the latest version of Azure CLI in Harness Delegate?
> 
> Add the below commands in the `INIT_SCRIPT` to download and install the Azure CLI for the delegate.
> 
> ```yaml
> - name: INIT_SCRIPT
>   value: |
>     rpm --import <https://packages.microsoft.com/keys/microsoft.asc>
>     rpm -ivh <https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm>
>     microdnf install -y azure-cli
> ```

Reviewer 2: ```suggestion
```

"Exit code 137" typically indicates an out-of-memory error. When a process in a system exhausts its allocated memory resources, the operating system sends a termination signal to the process. In the case of "Exit code 137," this signal signifies that the process was terminated due to running out of memory. This error commonly occurs when a program or container attempts to allocate more memory than is available, leading to termination by the system to prevent resource exhaustion and potential system instability.

### How to install the latest version of Azure CLI in Harness Delegate?

1. Locate the `INIT_SCRIPT` in the `Deployment` object of the `delegate.yaml` file.
   ```
   - name: INIT_SCRIPT  
    value: ""

>   ```

Reviewer: ```suggestion
   ```
```
